### ISSUE TITLE
Default value for accessibility level

### ISSUE THREAD

**Issue by srini-leanfolks** [paypal/AATT#66]
In Readme.md, the default value for accessibility level mentioned is 'WCAG2A'.

But in module, the default level is 'WCAG2AA'.

https://github.com/paypal/AATT/blob/70a152d4a5f89fce841f751813a137ba0f6048a9/module.js#L14

**Comment by mpnkhan** [comment]
That was a mistake in readme. Should be WCAG2AA only. Thanks for pointing it out.

**Comment by srini-leanfolks** [comment]
I never contributed to an open-source project. I thought of doing this but I didn't mentioned that part.
